src/libraries/DrawAccumulatorLib.sol

```diff
@@ -56,7 +56,7 @@ library DrawAccumulatorLib {
     /// @return True if a new observation was created, false otherwise.
     function add(Accumulator storage accumulator, uint256 _amount, uint32 _drawId, SD59x18 _alpha) internal returns (bool) {
         if (_drawId == 0) {
-            revert InvalidDrawId(_drawId);
+            revert AddToDrawZero();
```

Comment:
I can't get coverage on this line and I don't know why. I have a test for it that passes, but still no coverage...